src/index.js

```diff
@@ -1,9 +1,11 @@
+// import _ from 'lodash';
 import './style.css';
 import enterImg from './img/enter.png';
 import trashImg from './img/trash.png';
 import task from './modules/taskList.mjs';
 import listen from './modules/addtask.mjs';
 import newLi from './modules/taskBuild.mjs';
+import clear from './modules/clearComp.mjs';
 
 const todoList = document.getElementById('todoList');
 
```

Comment:
- [ ] As per the general documents, your project is supposed to be documented in a professional way. In your ReadMe, I cannot find the procedure to run you project. You seem to just have added the template and removed the Microverse Logo. Kindly update all the required sections of your ReadMe